src/battle_script_commands.c

```diff
@@ -6260,7 +6260,16 @@ static void Cmd_moveend(void)
             if (gMovesInfo[gCurrentMove].danceMove)
             {
                 u8 battler, nextDancer = 0;
+                bool8 turnOnHitmarker = FALSE;
 
+                for (battler = 0; battler < MAX_BATTLERS_COUNT; battler++)
+                    if (gSpecialStatuses[battler].dancerUsedMove)
+                    {
+                        // in case a battler fails to act on a Dancer proc
+                        turnOnHitmarker = TRUE;
+                        break;
+                    }
+                
```

Comment:
```suggestion
                for (battler = 0; battler < MAX_BATTLERS_COUNT; battler++)
                {
                    if (gSpecialStatuses[battler].dancerUsedMove)
                    {
                        // in case a battler fails to act on a Dancer proc
                        turnOnHitmarker = TRUE;
                        break;
                    }
                }
                
```